ovh/types_cloud_project_database.go

```diff
@@ -301,10 +305,13 @@ func (opts *CloudProjectDatabaseUpdateOpts) FromResource(d *schema.ResourceData)
 	if err != nil {
 		return nil, err
 	}
+	time := d.Get("backup_time").(string)
 
-	opts.Backups = CloudProjectDatabaseBackups{
-		Regions: regions,
-		Time:    d.Get("backup_time").(string),
+	if engine != "kafka" && (len(regions) != 0 || time != "") {
```

Comment:
not the same check as in `CloudProjectDatabaseCreateOpts.fromResource` ? (no `engine != "kafka"` there)

Reply:
`backups` field is ignored for a lot of engine in API (because the engine has not backup or backup not configurable) in POST and PUT, Kafka is the only engine where the schema of the PUT do not has "backups" fields. So this `engine != "kafka"` will ignore backups attribute for `Kafka` when update, no need to add this in CloudProjectDatabaseCreateOpts.fromResource because our API already ignore this field